ffac-eol-ssid/luasrc/lib/gluon/upgrade/950-eol-ssid-nagger

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/lua
+
+local util = require 'gluon.util'
+local site = require 'gluon.site'
+local uci = require('simple-uci').cursor()
+
+if site.eol_wifi() == nil then
+        -- Not applicable
+        os.exit(0)
+end
+
+
+-- Get/initialize enabled state
+if site.eol_wifi.ssid() == nil then
+        -- Not applicable
+        os.exit(0)
+end
+
+ssid = site.eol_wifi.ssid()
+
+if not uci:get('eol_wifi', 'ssid') then
+        uci:section('eol_wifi', 'ssid', 'ssid', {
+                enabled = true,
+                ssid = ssid,
+        })
+        uci:save('eol_wifi')
+end
+local enabled = uci:get_bool('eol_wifi', 'ssid', 'enabled')
+
+-- Change client radio ssid
+if enabled then
+        uci:set('wireless', 'client_radio0', 'ssid', ssid)
```

Comment:
inverted case missing?

Reply:
If I am not mistaken, the SSID changes on each upgrade to the value given in the site.
As this script is only run on upgrade, no further action is needed.

Reply:
These scripts are also run on domain-reconfiguration.

Reply:
Right, I am not used to multi domain setups - fixed it..
Or did I? in a multidomain setup the wifi ssid param might not be set in site but in its domain? what should I use then?

Reply:
Good point, I didn't think of this.

I'd suggest to rename the uci toggle from `enabled` to `disabled` and not set it from the site. This way, the package can be disabled via UCI.

`if enabled then` can be replaced to `if not uci_disabled  then`.

Once we are at this call, the site intents to enable the EOL-SSID.

Also a small nitpick to make it nicer: uci packages usually use dashes instead of underscores. `eol_wifi` --> `eol-wifi`.

Reply:
regarding the underscore:
the package is named correctly with dash, only the uci param has the underscore.
Printing `uci show` does not seem to have any guidelines:

`gluon-node-info.@location[0].share_location` 

or 

`gluon-setup-mode.@setup_mode[0]=setup_mode`

but the first part before the dot seems to use dashes and everything else uses underscore :shrug: 
so I will go with this guideline I extracted from it ;)

Reply:
> These scripts are also run on domain-reconfiguration.

but the wifiname is then reset correctly too (to the domain value) and not overwritten by the upgrade of this package if enabled is not set - so no change to the first version was needed for domain support?

changing enabled to disabled would not solve the problem we thought we had here too?
But I might be on some weird track too..

I cleaned up the configuration too